.github/workflows/publish-new-release.yml

```diff
@@ -38,20 +38,15 @@ jobs:
           git config user.name "GitHub actions"
           git config user.email noreply@github.com
 
-      - name: Create Github Release & Tag
+      - name: Create Github Release
         id: create_release
         env:
           HUSKY: 0
           GITHUB_TOKEN: ${{ secrets.PAT }}
           CONVENTIONAL_GITHUB_RELEASER_TOKEN: ${{ secrets.PAT }}
         run: |
-          git for-each-ref refs/tags
-          ./Scripts/find-tag.sh ${{ steps.extract-version.outputs.release_version }} && git tag --delete ${{ steps.extract-version.outputs.release_version }} && git push --delete origin ${{ steps.extract-version.outputs.release_version }}
-          git tag -a v${{ steps.extract-version.outputs.release_version }} -m "chore: release v${{ steps.extract-version.outputs.release_version }}"
-          git push origin refs/tags/v${{ steps.extract-version.outputs.release_version }}
-          git for-each-ref refs/tags
-          DEBUG=conventional-github-releaser npx conventional-github-releaser -p angular --config github-release.config.js
-
+          DEBUG=conventional-github-releaser npx conventional-github-releaser -p angular
```

Comment:
we do not need --config github-release.config.js? 

conventional-github-releaser will ignore tags on commit hash that is a merge commit other wise